.github/actions/test/action.yml

```diff
@@ -4,6 +4,10 @@ inputs:
   gateway-url:
     description: "The URL of the IPFS Gateway implementation to be tested."
     required: true
+  subdomain-url:
+    description: "The Subdomain URL of the IPFS Gateway implementation to be tested."
+    default: "http://example.com"
+    required: false
```

Comment:
I'll add a comment in the README, there's no real proxying anymore, we really use that value to overwrite the request sent to the gateway.

Reply:
I see. Thanks for adding the comment. It makes things clearer. Few more thoughts that it sparked with me.

Wouldn't it make sense to always use what we now call `subdomain-url`. As in, all the test should talk to `gateway-url` in a way that thinks it's called `subdomain-url`? Theoretically, wouldn't that be more correct? So really, every test case should be executed 3 times (with host header, proxy, proxy with tunneling) as long as `subdomain-url` and `gateway-url` are different.

Looking more closely at how we execute the test cases with host header, I think we're not using the protocol part of `gateway-url`. Is that correct?

If so, isn't what we really need from the user these two pieces of info:
- What are you calling/want to call your gateway? Please give us the full URL with a domain name. That would be https://dweb.link for example or http://example.com. You could give us http://127.0.0.1:8080 here as well BUT there's no way for you to support sudomains on that URL so you'll have to disable subdomain-gateway spec. 
- Where is your gateway hosted? Please give us the host we should contact when we're making requests to the URL you provided in the previous step. If you don't, we're just going to use the host from the URL. It's fine if you give us a host with IP or a host with domain. This would be 127.0.0.1 for example or 127.0.0.1:8080 or foo.bar.example.com.

That would come together to something like this - https://github.com/ipfs/gateway-conformance/compare/main...gateway-host (note that I intentionally skipped modifying test/config code; I just wanted to get the idea across). 

There's absolutely no rush with replying to this. I just want to understand what might be most straight-forward way for those running the tests to provide us all the information we need.